src/main/java/org/resalePlatform/controller/AdController.java

```diff
@@ -19,38 +21,38 @@ public AdController(AdService adService) {
         this.adService = adService;
     }
 
-    @GetMapping("/ads")
+    @GetMapping("/")
     public ResponseEntity<Ads> getAllAds() {
         return null;
     }
 
-    @PostMapping(value = "/ads", consumes = MediaType.MULTIPART_FORM_DATA_VALUE)
-    public ResponseEntity<Ads> addAd(@RequestBody Ads ad, @RequestParam("image") MultipartFile image) {
+    @PostMapping(value = "/", consumes = MediaType.MULTIPART_FORM_DATA_VALUE)
+    public ResponseEntity<Ads> addAd(@RequestBody Ads ad, @RequestPart("image") MultipartFile image) {
```

Comment:
```suggestion
    public ResponseEntity<Ads> addAd(@RequestPart("properties") Ads ad, @RequestPart("image") MultipartFile image) {
```
вот так должно быть, как сейчас работать не будет